web/src/pages/starter-kits/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { NavBar } from "@/components/navbar/NavBar";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { SingleColumnContainer } from "@/components/njwds/SingleColumnContainer";
+import { PageCircularIndicator } from "@/components/PageCircularIndicator";
+import { useRouter } from "next/router";
+import { ReactElement, useEffect } from "react";
+
+interface Props {
+  noAuth: boolean;
+}
+
+const StarterKitsRootPage = (): ReactElement => {
+  const router = useRouter();
+
+  useEffect(() => {
+    router.push("/starter-kits/generic");
+  }, [router]);
```

Comment:
[pebble] By doing it this way, we don't get the intended effect. We're using JavaScript-based client-side routing to get the user to the right page instead of using server-side rendering. The expected behavior would be that

```shell
curl -I --get "http://localhost:3000/starter-kits"
```

should return an HTTP 301 to the generic starter kit page rather than HTTP 200 which is what it gives now.